src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20240621123907_EES5170_AddBoundaryDataTable.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    /// <inheritdoc />
+    public partial class EES5170_AddBoundaryDataTable : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
I think you can add custom code a the end of this method to drop the `dbo.geojson` view.

You can also delete the `Current_View_geojson.sql` file in the `Migrations` folder.

I think we discussed some other stuff which can eventually be cleaned up at a later date like the `geometry2json` function and the geometry tables so it would be good to create a separate 'tidy-up' ticket for these.

Reply:
Created EES-5292 for the removals.